test_regression.py

```diff
@@ -0,0 +1,180 @@
+
+#%%
+import pytest
+from pathlib import Path
+import numpy as np
+import os
+
+from utils import extract_test_geometries, extract_scenario_parameters, execute_and_assert
+
+os.environ["ENDPOINT"] = "https://openeo.dataspace.copernicus.eu/"
```

Comment:
This hardcoded `os.environ["ENDPOINT"]` should not be here I guess

Reply:
would you place it in conftest?

Reply:
no this does not belong the project's source code like that, it has to be a runtime environment setting, specified in the jenkins/nifi launch command.

or when running manually, something liek this:
```
export ENDPOINT=openeo.dataspace.copernicus.eu
pytest ...
```


Sidenote: I would also pick a better name than `ENDPOINT` from the start,  e.g. use `OPENEO_BACKEND` instead